tslint.json

```diff
@@ -0,0 +1,86 @@
+{
+  "extends": ["tslint:recommended", "tslint-config-prettier"],
+  "rules": {
```

Comment:
Ideally we'd find a base config which doesn't need any rules to be overwritten.

Reply:
We could just extend `tslint:recommended`, `tslint-config-prettier`, and `tslint-microsoft-contrib`.

I changed a lot of rules that default to errors to warnings though because the conversion is a work in progress.